src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj

```diff
@@ -37,4 +37,6 @@
     <Content Remove="Properties\launchSettings.json" />
   </ItemGroup>
 
+  <Import Project="../../../Maui.InTree.props" Condition=" '$(UseMaui)' != 'true' " />
```

Comment:
Workloads put these files AFTER the csproj, so having them in the Directory.Build.props is different behaviour.